editor.planx.uk/src/@planx/components/SetValue/utils.test.ts

```diff
@@ -84,6 +84,7 @@ describe("calculateNewValues() helper function", () => {
         expected: ["bear", "dog", "monkey"],
       },
       { previous: ["bear", "dog", "lion"], expected: ["bear", "dog"] },
+      { previous: ["bear"], expected: ["bear"]},
```

Comment:
Previously, `["bear"]` was resulting in `"bear"` and falling under the same conditions as `"panda"` to `"panda"` above (because `previous` values get auto-formatted as an array)